src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/config/ExceptionHandler.kt

```diff
@@ -88,6 +88,14 @@ class SendLegalMailToPrisonsApiExceptionHandler {
       .body(ErrorResponse(status = CONFLICT, errorCode = DuplicateContact))
   }
 
+  @ExceptionHandler(ContactNotFoundException::class)
+  fun handleContactNotFoundException(e: ContactNotFoundException): ResponseEntity<ErrorResponse> {
```

Comment:
Any particular reason not to use the generic EntityNotFoundExceeption? Otherwise this class could get very large...